AdyenActions/Components/QRCode/QRCodeComponent.swift

```diff
@@ -7,6 +7,12 @@
 import Adyen
 import UIKit
 
+/// An error that occured during the use of QRCodeComponent
+internal enum QRCodeComponentError: Error {
```

Comment:
can you make it `LocalizedError` with  a description?

```
internal enum QRCodeComponentError: LocalizedError {
case qrCodeExpired

public var errorDescription: String? {
       "QR code is not longer valid"
    }
}
```

so that merchants can see this message when they print the error in the console.

Reply:
Yes. Was not aware of `LocalizedError` :)